test/scripts/e2e_subs/rest.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# TIMEOUT=300
+
+date '+rest.sh start %Y%m%d_%H%M%S'
+
+set -ex
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+gcmd="goal -w ${WALLET}"
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# REST Parameters
+PUB_TOKEN=$(cat "$ALGORAND_DATA"/algod.token)
+ADMIN_TOKEN=$(cat "$ALGORAND_DATA"/algod.admin.token)
+NET=$(cat "$ALGORAND_DATA"/algod.net)
+
+function call_admin {
+  curl -q -s -H "Authorization: Bearer ${ADMIN_TOKEN}" "$NET$1"
+}
+
+function call {
+  curl -q -s -H "Authorization: Bearer ${PUB_TOKEN}" "$NET$1"
+}
+
+# $1 - test description.
+# $2 - query
+# $3 - substring that should be in the response
+function call_and_verify {
+  local RES
+  RES=$(call "$2")
+  if [[ "$RES" == *"$3"* ]]; then
+    return 0
+  fi
+  echo "Failed test: $1"
```

Comment:
Could you further disambiguate the failed test when they may have the same name, ala "Should contain app data."?

Reply:
The script uses `set -x`, so all the output should be printed, so we should be alright even without this line. When the test passes it gets hidden by the framework

Reply:
ok